drivers/wifi/nrf700x/osal/fw_if/umac_if/inc/fmac_structs.h

```diff
@@ -452,6 +452,7 @@ struct wifi_nrf_fmac_dev_ctx {
 	bool fw_deinit_done;
 	bool alpha2_valid;
 	unsigned char alpha2[3];
+	bool twt_sleep_status;
```

Comment:
Use enum instead of bool to enhance code readability. E.g. enum twt_state where values can be TWT_STATE_ASLEEP, TWT_STATE_AWAKE.